requirements.txt

```diff
@@ -0,0 +1,21 @@
+llama-index==0.10.30
```

Comment:
is it in `pyproject.toml` now, right?

Reply:
yes

Reply:
so this file should be deleted from PR, right?

Reply:
@ilan-pinto forgot to push something?